circleguard/investigator.py

```diff
@@ -26,6 +28,8 @@ class Investigator:
     """
 
     MASK = int(Key.M1) | int(Key.M2)
+    # https://osu.ppy.sh/home/changelog/cuttingedge/20190111
+    VERSION_SLIDERBUG_FIXED = 20190111
```

Comment:
I asked MillhioreF and if MillhioreF is correct, the initial implementation of sliderbug fix was this version. Will have to do some testing later to confirm.

Reply:
only for cutting edge though? we should probably be using the first version it was fixed for stable (maybe with a comment to the first cutting edge changelog where it was fixed)

Reply:
stable implementation of this was 20190207, but the previous stable version before that is 20181231, so using cutting edge version number won't affect plays set on stable version 20181231.

Reply:
I see, makes sense

Reply:
hmm tough call with the versioning...I think people use circleguard more often with non-local replays, so we should probably err on the side of caution (ie 20190207) so we don't give the wrong ur for stable replays set between 20190111 and 20190207

Reply:
yeah I was thinking that too, <3% of players use cutting edge anyways. I could add an exception for local replays to use the cutting edge version, not sure if that's a good idea though.

Reply:
sure, that's probably fine. `Replay` subclasses are stable enough I'm not too concerned about relying on their implementation details (like having a concrete version or not)